src/main/java/com/sprint/mission/discodeit/entity/Channel.java

```diff
@@ -0,0 +1,98 @@
+package com.sprint.mission.discodeit.entity;
+
+import java.util.ArrayList;
+
+public class Channel extends BaseEntity
+{
+    private String channelName;
+    private User hostUser;
+
+    private ArrayList<Message> messages;
+    private ArrayList<User> users;
```

Comment:
list 는 선언과 동시에 빈리스트를 생성해주시는게 좋습니다.
```suggestion
    private ArrayList<Message> messages = new ArrayList<>();
    private ArrayList<User> users  = new ArrayList<>();
```